packages/bolt-connection/src/connection/connection.js

```diff
@@ -17,7 +17,7 @@
  * limitations under the License.
  */
 
-import { ResultStreamObserver, BoltProtocol } from '..bolt'
+import { ResultStreamObserver, BoltProtocol } from '../bolt'
```

Comment:
Deno doesn't like this `..folder` format. (Was this intentional, or just an oversight? Just wanted to flag in case it's a convention or deliberate usage of some feature I don't know about.)

Reply:
It was probably some typo or auto-complete issue. It was not deliberated.